go/libraries/doltcore/sqle/dprocedures/dolt_tag.go

```diff
@@ -40,6 +41,9 @@ func doDoltTag(ctx *sql.Context, args []string) (int, error) {
 	if len(dbName) == 0 {
 		return 1, fmt.Errorf("Empty database name.")
 	}
+	if err := branch_control.CheckAccess(ctx, branch_control.Permissions_Write); err != nil {
```

Comment:
Tags are separate from branches, no reason to protect afaict

Reply:
That's fair. Since tags are writing _something_ I decided to gate it, but I can remove the check for them.